docs/content/integrations/dbt/reference.mdx

```diff
@@ -532,6 +532,39 @@ def my_dbt_assets(context: AssetExecutionContext, dbt: DbtCliResource):
 
 ---
 
+## Emit dbt table schema as materialization metadata <Experimental />
+
+Dagster allows you to emit table schema [materialization metadata](/concepts/assets/software-defined-assets#recording-materialization-metadata), which includes the column names and data types of your materialized dbt models, seeds, and snapshots.
+
+Similar to dbt, you don't need to define this table schema up front in your dbt project. Instead, Dagster will automatically emit this metadata when you materialize your dbt assets.
```

Comment:
What exactly do you mean by "similar to dbt" here?

Reply:
+1